config/ember-try.js

```diff
@@ -154,6 +154,24 @@ module.exports = function() {
             }
           }
         },
+        {
+          name: 'with-ember-qunit@5',
+          npm: {
+            devDependencies: {
+              'ember-source': urls[0],
+              'ember-qunit': '^5.0.0',
+              'qunit': '~2.13.0',
+              '@ember/test-helpers': '^2.0.0',
+              'ember-qunit-nice-errors': null,
+              'ember-qunit-source-map': null,
```

Comment:
seems some of these guys breaks Qunit during the build. Before merging I'd like to figure out what's going on.